docs/ChangeLog/20191129/PR7277.md

```diff
@@ -0,0 +1,4 @@
+* Migrating `ACTION_LAYER_TAP_KEY()` entries in `fn_actions` to `LT()` keycodes
```

Comment:
Would you please move/rename this file to `docs/ChangeLog/20200229/PR7277.md`?

Do that and I'll merge this into `future` branch.